server/config/config.js.dist

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  dev: true,
```

Comment:
Das hier unbedingt in deine `/server/config/config.js` eintragen! Sonst funktioniert es bei dir wahrscheinlich erstmal nicht mehr (bzw. du müsstest jedes Mal `npm run build` laufen lassen).